src/controllers/paymentType.controller.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const Constants = require('../constants')
+const BaseController = require('./base.controller')
+const CookieService = require('../services/cookie.service')
+const Application = require('../models/application.model')
+const {CARD_PAYMENT, BACS_PAYMENT} = Constants.Dynamics.PaymentTypes
+
+module.exports = class PaymentTypeController extends BaseController {
+  async doGet (request, reply, errors) {
+    const pageContext = this.createPageContext(errors)
+    const authToken = CookieService.get(request, Constants.COOKIE_KEY.AUTH_TOKEN)
+    const applicationId = CookieService.get(request, Constants.COOKIE_KEY.APPLICATION_ID)
+    const application = await Application.getById(authToken, applicationId)
+
+    if (application.isSubmitted()) {
+      return reply
+        .redirect(Constants.Routes.ERROR.ALREADY_SUBMITTED.path)
+        .state(Constants.DEFRA_COOKIE_KEY, request.state[Constants.DEFRA_COOKIE_KEY], Constants.COOKIE_PATH)
+    }
+
+    if (request.payload) {
+      pageContext.formValues = request.payload
+    } else {
+      pageContext.formValues = {}
+    }
+
+    Object.assign(pageContext.formValues, {
```

Comment:
I think the use of Object.assign here has made the code less readable and has used additional lines of code.